iphone/TitaniumKit/TitaniumKit/Sources/API/TiViewProxy.m

```diff
@@ -213,7 +213,9 @@ - (void)add:(id)arg
 
   if (windowOpened) {
     // Turn on clipping because I have children
-    [[self view] updateClipping];
+    if ([[self view] respondsToSelector:@selector(updateClipping)]) {
+      [[self view] updateClipping];
+    }
```

Comment:
I'm pretty sure this does not fix the real issue. Primarily, because this used to work before and the error suggests that a wrong instance is bubbled up in the event

Reply:
@vijaysingh-axway I think Hans is right here. I did a quick debug session and it appears that `self.view` is holding a wrong value. Xcode debugger says it's `nil` so it may have been accidentally released and now holds an invalid pointer. Can you please double check what's going on?